integr8ly/integreatly-operator
# add release preparation script

# Description
https://issues.redhat.com/browse/DEL-219

Add a script to generate a new csv and edit appropriate files with the new version. 

The script takes in a SEMVER variable which it will use to define the version and tag ( if it exists ) and edit/create associated files.

## Type of change




## Checklist

## Verification

1. Run `make release/prepare` and verify that nothing changes
2. Edit SEMVER in the Makefile to change the rc number
3. Run `make release/prepare` and verify that the images in the csv have changed
4. Edit SEMVER in the Makefile to change the version number
5. Run `make release/prepare` and verify that the new csv has been created and the package, makefile and version.go are all referencing the new version
